healthcheck-api/src/main/java/com/storebrand/healthcheck/HealthCheckReportDto.java

```diff
@@ -83,6 +83,18 @@ public static class HealthCheckDto {
         public Optional<String> structuredData = Optional.empty();
         /** Status about running time, and any issues encountered while running this check */
         public RunStatusDto runStatus;
+        /**
+         * Each health check has its own background thread that updates the check status. This specifies the interval we
+         * wait between each time we perform an new health check. The default is to wait 10 minutes since the last check
+         * finished. Note that this is the base configuration, and the interval might be higher if the status is non-ok.
```

Comment:
It can also be higher if we run the health checks synchronously, because they will actually run on each request.